css/main.css

```diff
@@ -1,3 +1,152 @@
-/* Add your custom CSS here */
+/*Index CSS*/
+body {
+    color: black;
+    font-family: 'Satisfy', cursive;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+header,
+footer {
+    background-color:   rgb(238, 237, 243);
+
+}
+
+a {
+    padding: 15px;
+    text-decoration: none;
+    color: black;
+}
+
+.header-text {
+    font-size: 25px;
+    margin-left: -80px;
+}
+
+span {
+    font-weight: bolder;
+}
+
+.blue {
+    width: 20%;
+    margin-top: 10px;
+}
+
+#main-text {
+    width: 280px;
+    text-align: justify;
+}
+
+h2 {
+    font-size: 25px;
+}
+
+h1 {
+    font-size: 80px;
+}
+
+.resume{
+    border-radius: 8px;
+    background-color: blue;
+    color: white;
+    padding: 6px 21px;
+}
+
+.projects {
+    border-radius: 8px;
+    background-color: white;
+    padding: 6px 21px;
+}
+
+.resume {
+  transition-duration: 0.4s;
+}
+
+.resume:hover {
+  background-color: white;
+  color: black;
+}
+
+.projects {
+  transition-duration: 0.4s;
+}
+
+.projects:hover {
+  background-color: blue;
+  color: white;
+}
+
+.col-md-5 {
+    background-position: contain;
+}
+
+.card img {
+    padding-top: 10px;
+    align-self: center;
+    padding-bottom: 10px;
+}
+
+#idcard {
+    margin-left: 280px;
+}
```

Comment:
Don't use IDs

Reply:
This `margin-left` defeats the purpose of using Bootstrap for your layout. Can you use an `offset` class instead?
You also need to account for mobile - this margin-left is making that card disappear because there is no room for it.